config.toml

```diff
@@ -0,0 +1,74 @@
+checkUpdate = true
+debug = false
+interval = 6
+cleanInterval = 3_600
+banTime = 86_400
+timeout = 6
+logToFile = true
+
+clientType = ""
+clientURL = "http://qbittorrent"
```

Comment:
Although it is helpful to understand, it is different from the default empty value "" in config.

Reply:
Wrong value by mistake.